app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -488,11 +499,58 @@ private void setupContentAndSpoiler(NotificationViewData.Concrete notificationVi
             Spanned content = statusViewData.getContent();
             List<Emoji> emojis = statusViewData.getStatusEmojis();
 
-            Spanned emojifiedText = CustomEmojiHelper.emojifyText(content, emojis, statusContent);
+            if(contentCollapseButton != null && statusViewData.isCollapsible() && (notificationViewData.isExpanded() || !hasSpoiler)) {
```

Comment:
Why do you need to check for null?

Reply:
I was working on some experiments to reuse some code between notifications and timeline but it didn't work out due to different layouts and data holders. Didn't work out.